api/watchlist/index.mjs

```diff
@@ -0,0 +1,26 @@
+import Watchlist from '../../backend/models/Watchlist.mjs';
+import connectToMongoDB from '../../backend/db/dbConnect.mjs';
+
+await connectToMongoDB();
+
+export default async function handler(req, res) {
+  console.log('connected to MongoDB');
```

Comment:
To be consistent, maybe add the `connectToMongoDB` function within the handler as well:
```suggestion

export default async function handler(req, res) {
  await connectToMongoDB();
  console.log('connected to MongoDB');
```